src/senaite/microorganism/browser/content/microorganismfolder.py

```diff
@@ -167,7 +167,8 @@ def get_category_title(self, microorganism, default=None):
         category = microorganism.category
         if category:
             category = category[0]
-        return self.get_obj_title(category, default=default)
+            return self.get_obj_title(category, default=default)
+        return default
```

Comment:
Just a small style note:

I usually return the "good" case at the end and a default within the condition, e.g.

```python
def get_category_title(self, microorganism, default=None):
    """Returns the category title of the microorganism passed-in
    """
    category = microorganism.category
    if not category:
        return default
    return self.get_obj_title(category, default=default)
```

So that it get more clear what it *should* return.

Reply:
Thanks @ramonski . Will take this into consideration, yes!